src/client/datascience/interactive-window/nativeInteractiveWindow.ts

```diff
@@ -225,16 +225,11 @@ export class NativeInteractiveWindow implements IInteractiveWindowLoadable {
         }
     }
 
-    public async addCode(code: string, file: Uri): Promise<boolean> {
+    public async addCode(code: string, file: Uri, line: number): Promise<boolean> {
         await this.updateOwners(file);
-        await this.addNotebookCell(code);
+        const notebookCell = await this.addNotebookCell(code, file, line);
         try {
-            await this.commandManager.executeCommand('notebook.cell.execute', {
-                ranges: [{ start: this.notebookDocument.cellCount - 1, end: this.notebookDocument.cellCount }],
-                document: this.notebookDocument.uri,
-                autoReveal: true
-            });
-            return true;
+            return this.submitCodeImpl(code, file.fsPath, 0, false, notebookCell);
```

Comment:
I was actually having to look this up myself. But this try catch block now seems funny. If you're directly returning the promise from submitCodeImpl I don't believe that the try catch block is relevant anymore. Looks to me like a case of this here: https://stackoverflow.com/questions/38708550/difference-between-return-await-promise-and-return-promise/42750371#42750371